Server/JokesOnYou.Web.Api/Models/Response/SavedJokeReplyDto.cs

```diff
@@ -1,6 +1,10 @@
-﻿namespace JokesOnYou.Web.Api.Models.Response
+﻿using System;
+
+namespace JokesOnYou.Web.Api.DTOs
 {
     public class SavedJokeReplyDto
```

Comment:
I need some help with understanding the use case. 
From what i understand: as a user i can save jokes, so that i can later see all my saved jokes. 
Wouldnt we then want to return the joke itself aswell when we are getting all saved jokes?

Reply:
probably yes.